doc/source/whatsnew/v0.25.1.rst

```diff
@@ -25,6 +25,7 @@ Other enhancements
 Bug fixes
 ~~~~~~~~~
 
+- Bug in :meth:`DataFrame.read_excel` with `engine='ods'` when `sheet_name` argument references a non-existent sheet (:issue:`27676`)
```

Comment:
can you move this to the IO section, double backticks rather than single around ``engine='ods'`` and ``sheet_name``